modules/@apostrophecms/widget-type/ui/apos/mixins/AposWidgetMixin.js

```diff
@@ -22,17 +22,11 @@ export default {
   },
   data() {
     return {
-      rendered: '...',
-      playerOpts: null,
-      playerEl: null
+      rendered: '...'
     };
   },
   mounted() {
     this.renderContent();
-    this.playerOpts = apos.util.widgetPlayers[this.type] || null;
-  },
-  updated () {
```

Comment:
I couldn't find any invocation of this.

Reply:
this is a vue lifecycle hook, not something we invoke directly. You're sure it can go?

Reply:
Ahhh I missed that, but we always call `renderContent` on a change of value, so I think we're OK without it.